packages/bsky/src/api/app/bsky/feed/getAuthorFeed.ts

```diff
@@ -33,7 +33,11 @@ export default function (server: Server, ctx: AppContext) {
 
       const [result, repoRev] = await Promise.all([
         getAuthorFeed(
-          { ...params, viewer },
+          {
+            ...params,
+            includeSoftDeleted: auth.credentials.type === 'role',
```

Comment:
just checking, is this supposed to be the string 'role'?

Reply:
yep. the `credentials.type === 'role'` is equivalent to `credentials.triage === true`